api/PaymentRequest.json

```diff
@@ -534,10 +534,10 @@
               "version_added": "44"
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
Indeed, no `paymentAddress` or `requestId` in https://github.com/WebKit/WebKit/blob/main/Source/WebCore/Modules/paymentrequest/PaymentRequest.idl.